packit_service/worker/tasks.py

```diff
@@ -40,3 +53,56 @@
 @celery_app.task(name="task.steve_jobs.process_message")
 def process_message(event: dict, topic: str = None) -> Optional[dict]:
     return SteveJobs().process_message(event=event, topic=topic)
+
+
+@celery_app.task(
+    bind=True,
+    name="task.babysit_copr_build",
+    # 120 minutes
+    default_retry_delay=120 * 60,
```

Comment:
Why is this set to 120 minutes? `countdown` in the `retry()` bellow will override this to be 60 seconds, and when the task is sent, it's also 60 seconds.

Another note: isn't doing this work every minute too often? The cadence is very close to what we would get just listening to the message bus. I would go with 5 minutes, but I can imagine even 15 minutes would be fine. Longer times seem to be a better fit for what this functionality would try to achieve: pick up the leftovers.

How many times will this retry? [3](https://docs.celeryproject.org/en/stable/reference/celery.app.task.html#celery.app.task.Task.max_retries)?

Reply:
Oh, and how about making all these values part of the configuration, so that we don't have to touch the code, to change them?

Reply:
yeah, this is a good point

so I changed it to use the exponential backoff, current setup:

do the first check in 120s, then next one in 60s, next 120s, 240s, 480s, etc.

would that be okay?

(btw after we merge this, I'm stoked to see how it goes on stg and will fix all the issues - which should be much easier to do and review once we see it in action)

also, I set max retries to 7

thanks for such a thorough review